src/VisualStudio/IntegrationTest/Microsoft.VisualStudio.Extensibility.Testing/Microsoft.VisualStudio.IntegrationTestService.Dev18/source.extension.vsixmanifest

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+  <Metadata>
+    <Identity Id="Microsoft.VisualStudio.IntegrationTestService" Version="1.0.0" Language="en-US" Publisher="Microsoft" />
+    <DisplayName>Visual Studio Integration Test Service</DisplayName>
+    <Description>Integration test service.</Description>
+    <MoreInfo>https://github.com/Microsoft/vs-extension-testing</MoreInfo>
+    <License>License.txt</License>
+    <!-- Don't include the GettingStartedGuide because it makes a web browser pop up after installation -->
+    <!--<GettingStartedGuide>https://marketplace.visualstudio.com/items?itemName=SamHarwell.MouseFastScroll</GettingStartedGuide>-->
+    <ReleaseNotes>https://github.com/Microsoft/vs-extension-testing/releases/tag/1.0.0</ReleaseNotes>
```

Comment:
Should we drop the release notes? or can we just point it at a readme in Roslyn?